test/test_ops.py

```diff
@@ -345,6 +345,115 @@ def test_out(self, device, dtype, op):
         op(*sample.input, *sample.args, **out_kwargs)
         self.assertEqual(expected, out)
 
+    @ops([op for op in op_db if op.aliases])
+    def test_aliases_consistency_eager(self, device, dtype, op):
+        # Currently, there is a code duplication based on
+        # test_variant_consistency_eager.
+        # To refactor those methods later.
+        samples = op.sample_inputs(device, dtype, requires_grad=True)
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        for sample in samples:
```

Comment:
This is cool but this can probably just validate using the first item in samples, instead of all of them.